spec/acceptance/clock_spec.rb

```diff
@@ -75,5 +75,13 @@
 
       it { should_not have_run("Run at certain time") }
     end
+
+    context 'when skip_first_run is present' do
+      let(:end_time) { start_time + 25.hours }
+
+      let(:clock_opts) { super().except(:max_ticks).merge(end_time: end_time, tick_speed: 12.hours) }
+
+      it { should have_run("Run but not at start").once }
+    end
```

Comment:
This needs to be guarded so that this test is only run against versions of clockwork that have the `skip_first_run` option available. Unfortunately, right now that appears to only be in master, so we can't do something as simple as checking the version. 

The regression test suite is failing for all non-master versions of clockwork we support, and I don't want to lose that signal, especially for pre-release versions.